packages/core/src/decorators/ack/AckDecorator.test.ts

```diff
@@ -24,7 +24,7 @@ describe('Decorators | AckDecoratorExtension', () => {
   })
 
   test('transforms Json to AckDecorator class', () => {
-    const transformed = JsonTransformer.fromJSON({ '~please_ack': {} }, TestMessage)
+    const transformed = JsonTransformer.fromJSON({ '~please_ack': {} }, TestMessage, { validate: false })
```

Comment:
What kind of validation errors do you get? Maybe we can add the required fields so we have validation included with the tests. I think adding an `@type` and `@id` should be enough (+ the required fields that TestMessage defines)

Reply:
if i enable validation for 
```json
{
          '~please_ack': {},
          '@id': undefined,
          '@type': undefined,
        }
```
I get:
```json
{"children": [], "constraints": {"matches": "id must match /[-_./a-zA-Z0-9]{8,64}/ regular expression"}, "property": "id", "target": {"~please_ack": {}}}, {"children": [], "constraints": {"matches": "type must match /(.*?)([a-zA-Z0-9._-]+)\\/(\\d[^/]*)\\/([a-zA-Z0-9._-]+)$/ regular expression"}, "property": "type", "target": {"~please_ack": {}}}, {"children": [{"children": [], "constraints": {"isArray": "on must be an array", "isEnum": "each value in on must be a valid enum value"}, "property": "on", "target": {}}], "property": "pleaseAck", "target": {"~please_ack": {}}, "value": {}}
```

Reply:
I think using something like this should work: 

```ts
{
  "@id": "1234-1234-1234-1234",
  "@type": "https://didcomm.org/test-protocol/1.0/test-message",
  "~please_ack": {
    "on": ["RECEIPT"]
   }
}
```